### ISSUE TITLE
BIOS update?

### ISSUE THREAD

**Issue by nitrox710** [leocg/Hackintosh-Dell-G5-5590#21]
I was using ur guide but windows updated the bios, now i cant boot, it say's: oc failed to load configuration, what can i do?

**Comment by leocg** [comment]
Your CFGLock config was lost. You can boot changing 2 properties under Kernel->Quirks in order to boot macOS without disabling it:
AppleCpuPmCfgLock to YES or 1
AppleXcpmCfgLock to YES or 1

You can also follow guide https://github.com/leocg/Hackintosh-Dell-G5-5590#disabling-cfg-lock-optional to disable it again.

Please let me know if it works with new BIOS version.

**Comment by nitrox710** [comment]
nope, still the same problem, seems like it didnt like the new bios

**Comment by leocg** [comment]
Could you send a screenshot or log from the boot process?

Sometimes BIOS update reset some configurations, check if your BIOS have the same config as the Readme.md.

**Comment by nitrox710** [comment]
Hi, i'm so sorry i didn't hace time until today there was no problem with
the Last bios, was a problem with mi copy of the efi, i have a working WiFi
Bluetooth and everything if u are interested, i5 1650 laptop

**Comment by al-jabirr** [comment]
> [leocg](https://github.com/leocg)

> Please let me know if it works with new BIOS version.

Dear Sir, (leocg) First of all thank you, because of you I successfully installed my hackingts. And does it work properly in the new bios you said? Yes Alhamdulillah it is working well. I am using the official new bios update 1.21.1, And yes I was able to lock CFG and undervolt as well. Thank you so much for this guide, your hard work deserves your success.
